src/components/KnownVariantsList/KnownVariantsList.tsx

```diff
@@ -74,17 +80,31 @@ function selectPreviewVariants(
   return variants;
 }
 
-export const KnownVariantsList = ({
-  country,
-  samplingStrategy,
-  onVariantSelect,
-  selection,
-  wholeSampleSetState,
-}: Props) => {
+/**
+ * For KnownVariantsList, we don't need very old data, since convertKnownVariantChartData (in load-data.ts)
+ * will only take the latest 2 months. However since our data collection lags by a couple
+ * of weeks, we need to fetch slightly more here to ensure we have enough. Therefore,
+ * we use past 3 months (Past3M) as the date range across all API calls within KnownVariantsList,
+ * e.g. useWholeSampleSet, getPangolinLineages, loadKnownVariantSampleSets.
+ * @param country
```

Comment:
Don't think we need to add these in the comment, unless you use it with the IDE somehow? (But I think TS infers this stuff anyway)

Reply:
The @param stuff

Reply:
> The @param stuff

Ah they are auto generated whey I added the JSDoc by typing `/** enter` . I can remove them.